games-action/heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
```

Comment:
How about a bit more wordy DESCRIPTION? See the first 2 paragraphs of "About" at http://hota.sourceforge.net/
Hmm... is the 2nd paragraph correct?

Reply:
Edited